config/environments/production.rb

```diff
@@ -19,13 +19,12 @@
   config.action_mailer.delivery_method = :smtp
 
   config.action_mailer.smtp_settings = {
-      domain: 'heroku.com',
-      address: 'smtp.sendgrid.net',
-      port: '587',
-      user_name: ENV['SENDGRID_USERNAME'],
-      password: ENV['SENDGRID_PASSWORD'],
-      authentication: :plain,
-      enable_starttls_auto: true
+      domain: 'sandbox1170a8e2fdc64c469b86cdb076202d27.mailgun.org',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.